test/e2e/adminapi_cluster_list.go

```diff
@@ -23,13 +23,8 @@ var _ = Describe("[Admin API] List clusters action", func() {
 		ctx := context.Background()
 		resourceID := resourceIDFromEnv()
 
-		// TODO: add pagination support once https://github.com/Azure/ARO-RP/pull/859 lands.
-		//       Replace code below with testAdminClustersList(...). See example below.
 		By("requesting the cluster document via RP admin API")
-		var ocs []*admin.OpenShiftCluster
-		resp, err := adminRequest(ctx, http.MethodGet, "/admin/providers/Microsoft.RedHatOpenShift/openShiftClusters", nil, nil, &ocs)
-		Expect(err).NotTo(HaveOccurred())
-		Expect(resp.StatusCode).To(Equal(http.StatusOK))
+		ocs := adminClustersList(ctx, "/admin/providers/Microsoft.RedHatOpenShift/openShiftClusters")
 
 		By("checking that we received the expected cluster")
 		var oc *admin.OpenShiftCluster
```

Comment:
I think we can remove this as well: we have it in `testAdminClustersList`.

Reply:
Unresolving this conversation as it is still valid, I think.

Reply:
Only this point reimains: we can remove a bunch of duplicated lines. This `it` should look similar to other `it`s below in this file. Something like:

```go
	It("should be able to return list of all clusters with admin fields", func() {
		ctx := context.Background()
		resourceID := resourceIDFromEnv()
		subscriptionID := os.Getenv("AZURE_SUBSCRIPTION_ID")

		path := "/admin/providers/Microsoft.RedHatOpenShift/openShiftClusters"
		testAdminClustersList(ctx, path, resourceID)
	})
```